widgets/src/other/Paginator.js

```diff
@@ -0,0 +1,110 @@
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["../common/HTMLWidget","css!./Paginator"], factory);
+    } else {
+        root.Entity = factory(root.HTMLWidget);
+    }
+}(this, function (HTMLWidget) {
+    function Paginator() {
+        HTMLWidget.call(this);
+        this._class = "paginator";
+
+        this._tag = "div";
+        
+        this._pageNumber = 1
+        
+        this._onSelect;
+
+        this.np = 1;
```

Comment:
Personally I would prefer better variable names - and its even harder for me to read with a np + pn